test/e2e/tests/editor/python-scratch-file-execution.test.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { test, tags } from '../_test.setup';
+
+test.use({
+	suiteId: __filename
+});
+
+test.describe('Python Scratch File', { tag: [tags.WEB, tags.EDITOR, tags.WIN] }, () => {
+	test('Verify that lines in a python scratch file with magics can be executed', async function ({ app, python, runCommand }) {
+
+		const filename = 'Untitled-1';
+		await test.step('Create a new python scratch file with code and a magic', async () => {
+			await runCommand('python.createNewFile');
+			await app.workbench.editor.waitForTypeInEditor(filename, 'import pandas as pd');
+			await app.code.driver.page.keyboard.press('Enter');
+			await app.code.driver.page.keyboard.press('Enter');
+			await app.workbench.editor.waitForTypeInEditor(filename, 'pip install pyarrow');
```

Comment:
I believe we needed the `%` to correctly reproduce the problem

```suggestion
			await app.workbench.editor.waitForTypeInEditor(filename, '%pip install pyarrow');
```

Reply:
Thanks!  I missed that.  It's added now.